components/gitpod-protocol/src/protocol.ts

```diff
@@ -151,7 +151,7 @@ export interface UserFeatureSettings {
  * The values of this type MUST MATCH enum values in WorkspaceFeatureFlag from ws-manager/client/core_pb.d.ts
  * If they don't we'll break things during workspace startup.
  */
-export const WorkspaceFeatureFlags = { full_workspace_backup: undefined, fixed_resources: undefined };
+export const WorkspaceFeatureFlags = { full_workspace_backup: undefined, fixed_resources: undefined, pvc: undefined };
```

Comment:
Personally I'm a big fan of more descriptive names, especially if it's in an "interface". E.g., it will be impossible in the admin UI to tell what exactly "pvc" stands for without prior knowledge/lookup in the code.

Does it mean "persistent_volume_claim"? Can we use that as feature flag? :thinking:

Reply:
@geropl Sounds good. Even that name is not quite right, but I do agree it is better than just `PVC`.

Reply:
Personally, I'd even go further and name `workspaces_use_persistent_volume_claim_for_storage` as it's real fun deprecating a feature flag two years later after it was forgotten through team reshuffles etc :)